src/hocs/ec-with-sorting/ec-with-sorting.js

```diff
@@ -13,40 +13,41 @@ const withSorting = createHOCc({
       type: Array,
       default: () => [],
     },
+    defaultSortCycle: {
```

Comment:
I'd remove the word `default` from the name. We do not expect any other SortCycle to be passed in the future.

Reply:
Valid point

- [ ] Rename